### PR TITLE
add overall design changes

### PR DESCRIPTION
**Pull request by RickardssonJ**

### CODE REVIEW

**Comment by simon-scott-frankdigital** [comment]
@RickardssonJ

- Please provide a brief description.
- When making design changes please provide 'before' and 'after' images, so that we can inspect and verify the change.
- Please provide a reference to wither a ticket, or a screenshot of a conversation to motivate the changes.

File: src/styles/default.ts [hunk 1]
```diff
@@ -61,8 +61,8 @@ const defaultStyle = css`
   p,
   li {
     margin: 0.8rem 0;
-
     font-size: 1.6rem;
+    line-height: calc(16px * 1.5);
```
Comment (simon-scott-frankdigital) [inline src/styles/default.ts:65]: Why is this not just 18px?

File: src/styles/global.ts [hunk 2]
```diff
@@ -48,18 +48,23 @@ export const breakpoints = {
   sm: '768px' as const,
   md: '1024px' as const,
   lg: '1440px' as const,
+  customBp: '1217px' as const,
```
Comment (simon-scott-frankdigital) [inline src/styles/global.ts:51]: This seems unnatural. Either add a new key `custom` and add `lg: 1217px` to that one. Or add it as a non-cutom attribute. Is the breakpoint part of design or made to remedy som issue with the implementation?

File: src/styles/global.ts [hunk 3]
```diff
@@ -48,18 +48,23 @@ export const breakpoints = {
...
 }
```
Comment (simon-scott-frankdigital) [inline src/styles/global.ts:52]: Also I think you could just type the object `as const`, and not need to add it to each row.

File: src/styles/global.ts [hunk 4]
```diff
@@ -181,3 +190,15 @@ export const LearningMaterialListItem = styled.li`
...
 `
+
+export const FlexContainer = styled.div`
+  display: flex;
+  justify-content: center;
+
+  gap: 8rem;
+
+  ${mq.sm} {
+    display: flex;
```
Comment (simon-scott-frankdigital) [inline src/styles/global.ts:201]: don't need to repeat `display: flex`, right?

File: src/utils/useGetScreenSize.ts
Comment (simon-scott-frankdigital) [inline src/utils/useGetScreenSize.ts:1]: Please name the file the same as the exported function, since the file only exports one function.

**Review by simon-scott-frankdigital — CHANGES_REQUESTED** [review]
Minor changes requested.

File: src/styles/default.ts (hunk 1 above)
Comment (RickardssonJ) [reply src/styles/default.ts:65]: I had an ide of making it more dynamic, having the font-size as a variable and multiplying it by 1.5.
But there was a bit too much work for that in this time frame that we have.
I will remove the calculation.

File: src/styles/global.ts (hunk 2 above)
Comment (RickardssonJ) [reply src/styles/global.ts:51]: I will look at it.

The breakpoint is for making sure the break happens before the metadata container gets too close to the rest of the content.

File: src/styles/global.ts (hunk 4 above)
Comment (RickardssonJ) [reply src/styles/global.ts:201]: Yes, you are absolutely right. I will fix it.

**Review by simon-scott-frankdigital — APPROVED** [review]
⭐

**Pull request merged**